srml/balances/src/lib.rs

```diff
@@ -1151,3 +1137,55 @@ where
 		Self::total_balance(who).is_zero()
 	}
 }
+
+impl<T: Trait<I>, I: Instance> Module<T, I> {
+	fn transfer_inner(
+		transactor: &T::AccountId,
+		dest: &T::AccountId,
+		value: T::Balance,
+		existential_requirement: ExistenceRequirement,
```

Comment:
```suggestion
		existence_requirement: ExistenceRequirement,
```

or does it just make more sense like this to me? :D